app/models/whitehall_migration.rb

```diff
@@ -4,6 +4,6 @@ class WhitehallMigration < ApplicationRecord
   has_many :document_imports
 
   def check_migration_finished
```

Comment:
Sorry minor nitpick - but it'd be consistent if we used completed in this method name as well instead of finished so same word is used for the same thing.

Reply:
So this has reminded me of something, we may not want to merge this PR at all. The reason that we named this method `check_migration_finished` rather than `check_migration_completed` was because `completed` is an actual state. This method allows a few other states that imply that the migration has finished running and isn't just checking to see if something is `completed`.

Thinking we may want to change this pr to name the column something like `finished_at` to avoid confusion with the `completed` state.

Reply:
Ah yes - good thinking. If I'm right we don't actually have a state field on a migration so it wouldn't conflict with terminology there but if finished_at avoids any meaning problems then I think that's a better shout.